backend/python-functions/realtime_rt.py

```diff
@@ -118,6 +118,13 @@ def compute_r_t(historical_case_counts):
     case_df = pd.Series(historical_case_counts['cases'], index=historical_case_counts['dates'])
     case_df.index = pd.to_datetime(case_df.index)
     case_df.index.name = 'date'
+    case_df.sort_index(inplace=True)
```

Comment:
Is it worth it to add a test for this logic too? Such as, a list of increasing cases with dates that are out of order so when it is sorted it is no longer increasing